pkg/tree/builder/gentoo/gentoo_test.go

```diff
@@ -46,4 +47,133 @@ var _ = Describe("GentooBuilder", func() {
+		})
+	})
+
+	Context("Parse ebuild3", func() {
+		parser := &SimpleEbuildParser{}
+		pkgs, err := parser.ScanEbuild("../../../../tests/fixtures/parser/linux-sources-1.ebuild")
+
+		It("parses correctly deps", func() {
+			Expect(err).ToNot(HaveOccurred())
+			fmt.Println("PKG ", pkgs[0])
+			Expect(len(pkgs[0].GetRequires())).To(Equal(0))
+			Expect(pkgs[0].GetLicense()).To(Equal(""))
+			Expect(pkgs[0].GetDescription()).To(Equal("Virtual for Linux kernel sources"))
+		})
+	})
+
+	Context("Parse ebuild4", func() {
+		parser := &SimpleEbuildParser{}
+		pkgs, err := parser.ScanEbuild("../../../../tests/fixtures/parser/sabayon-mce-1.1-r5.ebuild")
+
+		It("parses correctly deps", func() {
+			Expect(err).ToNot(HaveOccurred())
+			fmt.Println("PKG ", pkgs[0])
+			Expect(len(pkgs[0].GetRequires())).To(Equal(2))
+			Expect(pkgs[0].GetLicense()).To(Equal("GPL-2"))
+			Expect(pkgs[0].GetDescription()).To(Equal("Sabayon Linux Media Center Infrastructure"))
+		})
+	})
+
+	Context("Parse ebuild5", func() {
+		parser := &SimpleEbuildParser{}
+		pkgs, err := parser.ScanEbuild("../../../../tests/fixtures/parser/libreoffice-l10n-meta-6.2.8.2.ebuild")
+
+		It("parses correctly deps", func() {
+			Expect(err).ToNot(HaveOccurred())
+			fmt.Println("PKG ", pkgs[0])
+			Expect(len(pkgs[0].GetRequires())).To(Equal(146))
+			Expect(pkgs[0].GetLicense()).To(Equal("LGPL-2"))
+			Expect(pkgs[0].GetDescription()).To(Equal("LibreOffice.org localisation meta-package"))
+		})
+	})
+
+	Context("Parse ebuild6", func() {
+		parser := &SimpleEbuildParser{}
+		pkgs, err := parser.ScanEbuild("../../../../tests/fixtures/parser/pkgs-checker-0.2.0.ebuild")
+
+		It("parses correctly deps", func() {
+			Expect(err).ToNot(HaveOccurred())
+			fmt.Println("PKG ", pkgs[0])
+			Expect(len(pkgs[0].GetRequires())).To(Equal(0))
+			Expect(pkgs[0].GetLicense()).To(Equal("GPL-3"))
+			Expect(pkgs[0].GetDescription()).To(Equal("Sabayon Packages Checker"))
+		})
+	})
+
+	Context("Parse ebuild7", func() {
+		parser := &SimpleEbuildParser{}
+		pkgs, err := parser.ScanEbuild("../../../../tests/fixtures/parser/calamares-sabayon-base-modules-1.15.ebuild")
+
+		It("parses correctly deps", func() {
+			Expect(err).ToNot(HaveOccurred())
+			fmt.Println("PKG ", pkgs[0])
+			Expect(len(pkgs[0].GetRequires())).To(Equal(2))
+			Expect(pkgs[0].GetLicense()).To(Equal("CC-BY-SA-4.0"))
+			Expect(pkgs[0].GetDescription()).To(Equal("Sabayon Official Calamares base modules"))
+		})
+	})
+
+	Context("Parse ebuild8", func() {
+		parser := &SimpleEbuildParser{}
+		pkgs, err := parser.ScanEbuild("../../../../tests/fixtures/parser/subversion-1.12.0.ebuild")
+
+		It("parses correctly deps", func() {
+			Expect(err).ToNot(HaveOccurred())
+			fmt.Println("PKG ", pkgs[0])
+			Expect(len(pkgs[0].GetRequires())).To(Equal(25))
+			Expect(pkgs[0].GetLicense()).To(Equal("Subversion GPL-2"))
+			Expect(pkgs[0].GetDescription()).To(Equal("Advanced version control system"))
+		})
+	})
+
+	/*
+		Context("Parse ebuild9", func() {
```

Comment:
You can mark the test case with Pit, and make it skip (without needing to comment it out): https://onsi.github.io/ginkgo/ (see Pending specs)